Lesson_8_AsyncWeatherChecker/Program.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Spectre.Console;
+
+class Program
+{
+    const string apiKey = "45c6b1c77ec6da196fd2a2b805e63a78";
+
+    static async Task Main(string[] args)
+    {
+        string[] cities =
```

Comment:
Try to find some public API to get cities. Or you can check the existing weather API - whether it provides an endpoint for getting cities or not